pkg/skaffold/deploy/util/util_test.go

```diff
@@ -79,13 +79,9 @@ func TestRunContext_UpdateNamespaces(t *testing.T) {
 	}
 	for _, test := range tests {
 		testutil.Run(t, test.description, func(t *testutil.T) {
-			runCtx := &RunContext{
-				Namespaces: test.oldNamespaces,
-			}
+			ns := ConsolidateNamespaces(test.oldNamespaces, test.newNamespaces)
```

Comment:
nit: Should we rename the test function to reflect `ConsolidateNamespaces()`?

Reply:
copy paste error 😅  thanks